nsxt/utils_test.go

```diff
@@ -102,6 +102,10 @@ func getTestCertificateName(isClient bool) string {
 	return os.Getenv("NSXT_TEST_CERTIFICATE_NAME")
 }
 
+func getTestSitePath() string {
+	return os.Getenv("NSXT_TEST_SITE_PATH")
```

Comment:
We already have NSXT_TEST_SITE_NAME variable, so I suggest to define nsxt_policy_site data source based on the var in the test, and use path from it as an input to TZ data source. This will also validate the suggested usage.

Reply:
Sure. Will fix. Thanks.